keys/saml_test_sp.key

```diff
@@ -1,27 +1,27 @@
 -----BEGIN RSA PRIVATE KEY-----
```

Comment:
this key was changed in 1f8ed6da47ddb474934ed44bc85b31740468dfe2 and this PR reverts it. The key in 1f8ed6da47ddb474934ed44bc85b31740468dfe2 did not match the cert.